src/Api/Monetization/Controller/XAcceptedRatePlanController.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Apigee\Edge\Api\Monetization\Controller;
+
+use Apigee\Edge\Api\Monetization\Entity\XAcceptedRatePlanInterface;
+use Apigee\Edge\Api\Monetization\Entity\RatePlanInterface;
+use Apigee\Edge\Api\Monetization\Serializer\XAcceptedRatePlanSerializer;
+use Apigee\Edge\ClientInterface;
+use Apigee\Edge\Controller\EntityListingControllerTrait;
+use Apigee\Edge\Controller\EntityLoadOperationControllerTrait;
+use Apigee\Edge\Serializer\EntitySerializerInterface;
+use DateTimeImmutable;
+use Psr\Http\Message\UriInterface;
+use ReflectionClass;
+
+abstract class XAcceptedRatePlanController extends OrganizationAwareEntityController implements XAcceptedRatePlanControllerInterface
```

Comment:
Instead of this "X" prefix in class names, should not we leverage namespaces and open a new `Apigee\Edge\Api\ApigeeX` namespace to group implementations that only works on Apigee X? This would also help downstream devs to understand the complexity that they have to deal with.
I think this also relates to #115 somehow

(ps.: `Apigee\Edge\Api\X\... `looks odd to me but also an alternative...)

Reply:
@mxr576 thanks for bringing this up, we will use your suggestion of creating a `Apigee\Edge\Api\ApigeeX` namespace.